Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM mhart/alpine-node:6.10.3
+FROM mhart/alpine-node:8.2.1
 
-RUN apk add --no-cache git bash wget curl nano
-RUN npm install --global yarn@v0.24.5 --no-progress
+RUN apk add --no-cache git bash wget curl nano python make g++
```

Comment:
Do we really need python make g++ there?
The size of container more than doubled.

![dorbel_alpine-node_-_docker_hub](https://user-images.githubusercontent.com/468590/28822168-e6a46800-76c0-11e7-85fb-7149c5a9a8a6.png)